src/AttributesBuilder.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\HtmlBuilder;
+
+/**
+ * Builds tag attribute strings.
+ */
+class AttributesBuilder
+{
+    /**
+     * Builds arguments to a valid attributes string.
+     *
+     * @param array $attributes
+     *
+     * @return string
+     */
+    public function build (array $attributes) : string
+    {
+        $segments = [];
+
+        foreach ($attributes as $key => $value)
+        {
+            if (null === $value || false === $value)
+            {
+                continue;
+            }
+
+            if (true === $value)
+            {
+                $segments[] = $key;
+                continue;
+            }
+
+            $segments[] = \sprintf(
+                '%s="%s"',
+                $key,
```

Comment:
Shouldn't we escape the keys as well? Right now I could create ridiculous keys and break our normalization process and inject arbitrary contents.

Reply:
There is no way to escape keys, so that the result isn't horribly broken.
We could however reject invalid keys.

Reply:
See: https://github.com/Becklyn/html-builder/pull/2